src/primitives/transaction.h

```diff
@@ -241,7 +243,7 @@ class CTransaction {
 
     bool IsNull() const { return vin.empty() && vout.empty(); }
 
-    const TxId GetId() const { return TxId(hash); }
+    const TxId GetId() const { return TxId(id); }
```

Comment:
There's a static assert below for x86_64 only for some reason.  Probably needs to be bumped to 120 from 88. Although I am curious as to what that static assert is for exactly.

Reply:
Yeah let's just bump it.